build.gradle.kts

```diff
@@ -140,6 +142,31 @@ subprojects {
 		}
 
 		afterEvaluate {
+			if (project != project(":junit-jupiter")) {
+				tasks["jar"].withConvention(aQute.bnd.gradle.BundleTaskConvention::class) {
```

Comment:
Will this work for subprojects that have disabled the `jar` task and use `shadowJar` instead?

Reply:
That's a good question. I only briefly looked at `shadowJar` but I wasn't clear it's real purpose. Now that you mention it I'll have to dig deeper into it. If you could fill me in with a small description of what the intention is there it'll help me make the best choice on how to handle it.

Reply:
We use the shadow plugin to build a fat jar with relocated dependencies, e.g. in `junit-jupiter-params`. You could probably use the (not very pretty) approach from here to catch both tasks: https://github.com/junit-team/junit5/blob/master/buildSrc/src/main/kotlin/java-library-conventions.gradle.kts#L118

Reply:
k, this is solved.